exampleCourse/questions/element/figure/info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "uuid": "73f032f9-19de-41a6-ae9d-148d1c5264a3",
+    "title": "Element pl-figure: Display a statically or dynamically generated image",
```

Comment:
Can we add this question to the HW3 assessment (PrairieLearn element example gallery)?

Reply:
Added 👍